flowsettings.py

```diff
@@ -14,6 +15,14 @@
 # change this if your app use a different name
 KH_PACKAGE_NAME = "kotaemon_app"
 
+KH_APP_VERSION = None
```

Comment:
@lone17 I think we can also optionally allow people to specify APP_VERSION. So the priority would be:

1. Set by environment variable (e..g APP_VERSION)
2. version(KH_PACKAGE_NAME)
3. None